lib/grunt.js

```diff
@@ -39,11 +46,15 @@ module.exports = function(grunt) {
       htmlDest: 'preview.html',
       htmlTemplate: './templates/html.hbs',
       html: true,
-      rename: function(path) {
-        path = path.replace(svgRootDir, '');
+      rename: function(iconPath) {
+        // We just want the filename, so split on backslashes and pop the last one.
```

Comment:
This in in the category of pedantic nitpicks but now this comment is inaccurate.